HCCGo/package.json

```diff
@@ -23,14 +22,17 @@
     "diskusage": ">= 0.1.5",
     "electron-debug": "^1.0.1",
     "fs": ">= 0.0.2",
+    "electron-debug": "^1.0.1",
```

Comment:
Accident on my part! Was seeing if keeping that in devDependencies would help reduce total file size, but it caused build issues. Maybe worth looking into again at some point, but for now this is a 'whoops' on my part, I'll commit a fix for this.